### ISSUE TITLE
A question about database resets

### ISSUE THREAD

**Issue by schickm** [trane-project/trane-cli#11]
I think that my `practice_stats.db` got reset (all rows deleted...or tables dropped and rebuilt..not sure which) right after I upgraded from 0.2.1 to 0.2.7.  Is that expected behavior and should I always backup my `.trane/` directory prior to upgrading?  Or to phrase the question another way - should I be expecting that upgrading will require me to reset practice stats between versions?

I realize this is super beta stuff, so please don't feel like I'm asking for something here, I'm just trying set my own expectations.

Thanks for the awesome little app!  I've already started writing my own courses specific to learning major scale triad shapes on different strings!

**Comment by martinmr** [comment]
Not expected behavior, but best to back up your `.trane` folder just in case. The only changes to the database were in 0.2.0 which was a breaking change and in the just released 0.2.8 which just adds one index and does not touch the tables.

I'll see if I can reproduce the issue. I am using GitHub to keep track of my `.trane` folder, so that's another option for backups. The db files are binary files, so not the best choice for git, but it's free backup if you use GitHub.

**Comment by martinmr** [comment]
Yeah, I was able to reproduce it on my own database. No idea why because nothing changed in that database until v0.2.8, which just adds an index. I'll try to dig in further to make sure it's not something that happens in the future.

**Comment by martinmr** [comment]
Actually, I don't think I can reproduce it. I forgot I reset my DB myself when I upgraded from v0.1 to v0.2, which is why I wasn't seeing any scores. I installed trane-cli v0.2.1, opened the DB, added some dummy scores, and then upgraded to v0.2.7 and v0.2.8 each, and I was still seeing the records.

For now, it'd be better to keep backups just in case.

The newest versions have a `scores` command which should make it easier to check if there are previous recorded scores for an exercise.
